### PR TITLE
Include runtime dependencies in Nix package

### PR DESCRIPTION
**Pull request by lafrenierejm**
I had thought that these dependencies were already being propagated through from their inclusion as `buildInputs`, but it turns out they were not. The reason I hadn't noticed is because I had most of the packages installed independently, so in my day-to-day usage I didn't see any errors. 🤦

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -1,75 +1,117 @@
...
+
+        runtimeInputs = with pkgs; [
+          ffmpeg
+          imagemagick
+          pandoc
+          poppler_utils
+          ripgrep
+          tesseract
+          zip
+        ];
```
Comment (phiresky) [inline flake.nix:69]: tesseract and imagemagick are not used anymore

Comment (lafrenierejm) [reply flake.nix:69]: Will remove them. Thanks!

Comment (lafrenierejm) [reply flake.nix:67]: Removed in 4e8dd8f7a0a7d9091e088bf89c61ce6e24707864.

File: flake.nix [hunk 2]
```diff
@@ -1,75 +1,115 @@
...
-
     crane = {
       url = "github:ipetkov/crane";
       inputs.nixpkgs.follows = "nixpkgs";
     };
-
-    flake-utils.url = "github:numtide/flake-utils";
-
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    flake-root.url = "github:srid/flake-root";
```
Comment (phiresky) [inline flake.nix:11]: not really an expert in nix but it seems a bit weird to pull in some random code just to make this package (which is not very special) work

Comment (lafrenierejm) [reply flake.nix]: That's fair. IMO the couple extra inputs are worth it to make the flake more featureful and easier to maintain, but they're definitely not mandatory. I've pushed 81d04fb8e91fa277ed0b7a7dca8e7a53d0da2c5f without those additional inputs and the corresponding refactor.

**Comment by lafrenierejm** [comment]
@phiresky Any opinions on this PR?

**Pull request merged**